site/_includes/footer.html

```diff
@@ -1,4 +1,4 @@
-<footer role="contentinfo">
+<footer>
```

Comment:
No, it's redundant. You can always run the validator against the live site
to see the warning.

Ideally you should have the vnu checker but I'm not sure there is a gem nor
can I help integrating...

On Jul 21, 2016 01:17, "Parker Moore" <notifications@github.com> wrote:

> In site/_includes/footer.html
> <https://github.com/jekyll/jekyll/pull/5118#discussion_r71616819>:
>
> > @@ -1,4 +1,4 @@
> > -<footer role="contentinfo">
> > +<footer>
>
> Are you saying then that this is not a valid attribute?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jekyll/jekyll/pull/5118/files/b6044aebd610d2cce521ef81256703e839d9b19d#r71616819>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVVtcUCe21zSEorYfTgUl9mw2TOtfM3ks5qXp5ogaJpZM4JPZ_0>
> .
>